writethedocs/www
# Add devcontainer config option for development

### What
- Adds dev container config for easy development.
- In addition to local development with Python `venv`, this PR adds the Docker dev container option.

### Requirements
- Docker
- A [supported IDE](https://containers.dev/supporting#editors)

### How to test
- Checkout this branch.
- Open a [supported IDE](https://containers.dev/supporting#editors).
- Click the "Open in devcontainer" popup.
- Run `cd docs/ && make livehtml` for example to start the project.
- Click the "Open in Browser" popup.
- Project should start like normal on [http://127.0.0.1:8888](http://127.0.0.1:8888)
- Happy developing!


----
📚 Documentation preview 📚: https://writethedocs-www--2272.org.readthedocs.build/
